src/Redux/Book/Book.js

```diff
@@ -1,57 +1,55 @@
-import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable */
 
-const ADDBOOK = './bookStore/Book/ADDBOOK';
-const REMOVEBOOK = './bookStore/Book/REMOVEBOOK';
+// define constant
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const getBook = 'bookstore/books/getBook';
+export const fetchBooks = () => async (dispatch) => {
```

Comment:
Kindly make sure that you use createAsyncThunk in your project for creating thunk action creators so that you meet the requirements 🤓

Reply:
done